[2023-04-14T19:03:21.812+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T18:00:00+00:00 [queued]>
[2023-04-14T19:03:21.849+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T18:00:00+00:00 [queued]>
[2023-04-14T19:03:21.852+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T19:03:21.854+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-14T19:03:21.856+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T19:03:21.900+0000] {taskinstance.py:1309} INFO - Executing <Task(BranchPythonOperator): choose_endpoint_Odesa> on 2023-04-14 18:00:00+00:00
[2023-04-14T19:03:21.911+0000] {standard_task_runner.py:55} INFO - Started process 15482 to run task
[2023-04-14T19:03:21.918+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'choose_endpoint_Odesa', 'scheduled__2023-04-14T18:00:00+00:00', '--job-id', '584', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpi6qav77c']
[2023-04-14T19:03:21.921+0000] {standard_task_runner.py:83} INFO - Job 584: Subtask choose_endpoint_Odesa
[2023-04-14T19:03:22.113+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T18:00:00+00:00 [running]> on host ffea11fb791c
[2023-04-14T19:03:22.359+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=choose_endpoint_Odesa
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T18:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T18:00:00+00:00
[2023-04-14T19:03:22.364+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:313: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2023-04-14T19:03:22.366+0000] {python.py:177} INFO - Done. Returned value was: retrieve_historical_weather_data_Odesa
[2023-04-14T19:03:22.367+0000] {python.py:211} INFO - Branch callable return retrieve_historical_weather_data_Odesa
[2023-04-14T19:03:22.370+0000] {skipmixin.py:155} INFO - Following branch retrieve_historical_weather_data_Odesa
[2023-04-14T19:03:22.373+0000] {skipmixin.py:211} INFO - Skipping tasks ['retrieve_weather_data_Odesa']
[2023-04-14T19:03:22.731+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=choose_endpoint_Odesa, execution_date=20230414T180000, start_date=20230414T190321, end_date=20230414T190322
[2023-04-14T19:03:22.901+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-14T19:03:23.020+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
